rspace/src/main/scala/coop/rchain/rspace/RSpace.scala

```diff
@@ -2,27 +2,25 @@ package coop.rchain.rspace
 
 import java.nio.ByteBuffer
 
-import scala.collection.immutable.Seq
-import scala.concurrent.ExecutionContext
-import scala.util.Random
 import cats.effect.{Concurrent, ContextShift}
 import cats.implicits._
-import coop.rchain.shared.Log
+import com.typesafe.scalalogging.Logger
 import coop.rchain.catscontrib._
+import coop.rchain.metrics.Metrics
+import coop.rchain.metrics.Metrics.Source
 import coop.rchain.rspace.history.Branch
 import coop.rchain.rspace.internal._
 import coop.rchain.rspace.trace._
+import coop.rchain.shared.Log
 import coop.rchain.shared.SyncVarOps._
-import com.typesafe.scalalogging.Logger
-import coop.rchain.metrics.Metrics
 import org.lmdbjava.Txn
 import scodec.Codec
 
-object A {
```

Comment:
Where did this come from???

Reply:
I think @dzajkowski was feeling rebellious when he last modified RSpace.

Reply:
yep, my bad.